.github/workflows/release-spec.yml

```diff
@@ -184,10 +184,20 @@ jobs:
         branch: ${{ inputs.branch }}
         foascli_version: ${{ inputs.foascli_version }}
 
-  failure-handler: 
-    name: Failure Handler
+  retry-handler:
     needs: [ release, release-postman, release-changelog]
-    if: ${{ always() && contains(needs.*.result, 'failure') }}
+    if: ${{ always() && contains(needs.*.result, 'failure') && fromJSON(github.run_attempt) < 3}}
```

Comment:
nit: does GH actions not have some standard support for retries? would be nice if we didn't have to maintain our own retry logic

Reply:
There are some 3rd party Actions but they don't allow to restart a complete worflow. Let me know if you find something! we can always update this logic